### PR TITLE
LPS-149118

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related Issue: https://issues.liferay.com/browse/LPS-149118

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/object/object-service/service.xml [hunk 1]
```diff
@@ -526,6 +526,7 @@
 
 		<column name="objectViewId" type="long" />
 		<column name="objectFieldName" type="String" />
+		<column localized="true" name="label" type="String" />
```
Comment (gabrielwas) [inline modules/apps/object/object-service/service.xml:529]: @p-albuquerque can you sort this? Right after `objectFieldName`

Reason: First, we have audit fields... then ids... then custom fields.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java [hunk 2]
```diff
@@ -238,6 +238,7 @@ public ObjectView updateObjectView(
 				newObjectViewColumn.setObjectFieldName(
 					objectViewColumn.getObjectFieldName());
 				newObjectViewColumn.setPriority(objectViewColumn.getPriority());
+				newObjectViewColumn.setLabelMap(objectViewColumn.getLabelMap());
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java:241]: @p-albuquerque can you sort this too? Put right after `newObjectViewColumn.setObjectViewId`

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by liferay-forms** [comment]
Hey @p-albuquerque, can you please check if these failures are related to your changes? Thanks

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @p-albuquerque, can you please check if these failures are related to your changes? Thanks

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:forward

**Pull request closed without merging**